Code/Framework/AzCore/Tests/Jobs.cpp

```diff
@@ -23,6 +23,7 @@
 #include <AzCore/std/containers/array.h>
 #include <AzCore/std/containers/fixed_list.h>
 #include <AzCore/std/containers/unordered_set.h>
+#include <AzCore/std/parallel/binary_semaphore.h>
```

Comment:
Note, this header is needed for some of the tests, but no longer included in `JobManagerWorkStealing.h`